lib/pages/home.dart

```diff
@@ -203,35 +193,52 @@ class _HomePageState extends State<HomePage> {
         // Make sure that the user location request has been completed before creating the GoogleMap widget
         // Otherwise, the map widget will not be created properly on iOS
         body: FutureBuilder(
-          future: locationFuture,
-          builder: (BuildContext context, AsyncSnapshot<LocationData> snapshot) {
-            if (snapshot.connectionState == ConnectionState.waiting){
-              return Container();
-            } else {
-              return GoogleMap(
-                onCameraMove: (position) {
-                  if (_cameraPosition.zoom != position.zoom) {
-                    setState(() {
-                      _displayedMarkers = fluster
-                          .clusters([-180, -85, 180, 85], _cameraPosition.zoom.floor())
-                          .map((ClusterableMapMarker cluster) => cluster.toMarker())
-                          .toSet();
-                    });
-                  }
-
-                  _cameraPosition = position;
-                },
-                initialCameraPosition: _cameraPosition,
-                markers: _displayedMarkers,
-                myLocationButtonEnabled: false,
-                myLocationEnabled: true,
-              );
-            }
+          future: _garageFuture,
+          builder: (BuildContext context, AsyncSnapshot<List<Garage>> snapshot) {
+            return GoogleMap(
+              onCameraMove: (position) {
+                if (_cameraPosition.zoom != position.zoom) {
+                  setState(() {
+                    _displayedMarkers = fluster
+                        .clusters([-180, -85, 180, 85], _cameraPosition.zoom.floor())
+                        .map((ClusterableMapMarker cluster) => cluster.toMarker())
+                        .toSet();
+                  });
```

Comment:
indentation (4 spaces vs. 2 spaces)?

the indentation makes it a bit difficult to read since it doesn't line up with the `fluster` variable. If the lines are far too long if you line it up, then I would leave it as is.